lib/pbench/agent/tool_meister.py

```diff
@@ -1044,7 +1044,7 @@ def _send_client_status(self, status: str) -> int:
                 ret_val = 0
         return ret_val
 
-    def _create_tool_directory(self, directory: str) -> Path:
+    def _create_tool_directory(self, directory: str) -> Tuple[Path, Path]:
```

Comment:
I wish GitHub would allow comments on unchanged lines...

But this change in the signature is not reflected in the docstring, which says that it "Returns a Path object [...]". That should be updated to reflect reality.